react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
There is an `outline: hidden` being applied to the `.prism-code` that wasn't in play with the old `TabGate` code. The `TabGate` had a wrapper and that is where we were outlining before, now it's directly on the `Editor` with it's existing css.